contacts/emails_method_setting.py

```diff
@@ -0,0 +1,38 @@
+# gmail settings
+EMAIL_HOST = 'smtp.gmail.com'
```

Comment:
Read up on mailtrap.io and use there SMTP credentials.

Reply:
no this is unnecessary file by mistake push sorry